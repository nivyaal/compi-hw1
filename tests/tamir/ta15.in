// unknown token

int power = 5 ^ 5;