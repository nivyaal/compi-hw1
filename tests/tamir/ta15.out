1 COMMENT //
3 INT int
3 ID power
3 ASSIGN =
3 NUM 5
Error ^
